### DISCUSSION TITLE
Windows installation question

### DISCUSSION THREAD

**Discussion by ghost** [uiua-lang/uiua#228] [General]
Please forgive a naive question about installation on Windows 10.
I downloaded the latest version (0.1) and clicked on the exe file.
I continued past the Windows Defender SmartScreen by clicking Run Anyway.
A console window opened for an instant and then closed. 
I had expected uiua to be an interpreter that would stay open and allow user interaction.  The quick open and close of the console window is something I encounter with compilers that run source files.
Am I mistaken about uiua being an interpreter or is there something I can do to keep the console window open?
Thank you!
Carey

**Comment by kaikalii** [comment]
You should put it in you PATH and run it from a terminal.

**Reply by ghost** [reply]
Thank you!

**Reply by auracapo** [reply]
Could you explain how to put it in the PATH, and how to run it in the terminal?

**Reply by kaikalii** [reply]
On Windows, you need to go to your environment variables (search it) and edit `PATH` to have an entry for the folder `uiua.exe` is in.
Then just open any terminal (`cmd`, PowerShell, Windows Terminal), etc, and type `uiua`.

**Reply by auracapo** [reply]
I created a folder below the python folder where the PATH is and added it to the PATH where Uiua is with the executable, but it didn't work.

I also ran the uiua command in cmd and it was unable to read.

Am I doing something wrong? I tried searching, I tried Bard, GPT-4 and I couldn't find the solution.

**Reply by kaikalii** [reply]
Have a screenshot of it in your PATH?

**Reply by auracapo** [reply]
![image](https://github.com/uiua-lang/uiua/assets/114449444/a7fc065e-cabd-49b3-8a9d-25f624f30276)

![image](https://github.com/uiua-lang/uiua/assets/114449444/ab485eea-d183-4963-be82-b01ba9837329)

**Reply by kaikalii** [reply]
You'll have to rename the exe to uiua.exe.

**Reply by auracapo** [reply]
Oh, thank you it worked!!!!
